raw_socket_listener/tcp_message.go

```diff
@@ -92,9 +92,14 @@ func (t *TCPMessage) IsMultipart() bool {
 	}
 
 	payload := t.packets[0].Data
-	m := payload[:4]
```

Comment:
I see. Seems I've misunderstood the packet reconstruction process. Ok, will change it to the `if len(payload) < 4` solution.